docs/windows.md

```diff
@@ -106,6 +106,7 @@ Remember that [volume mounts on Windows can only target a directory](https://kub
 ## Limitations
 
 - Sharing process namespaces [doesn't work on Windows](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#v1-pod) so you can't use the Process Namespace Sharing (pns) workflow executor.
+- The argoexec Windows container is built using [nanoserver:1809](https://github.com/argoproj/argo-workflows/blob/master/Dockerfile.windows#L28) as the base image. Running a newer windows version (e.g. 1909) is currently [not confirmed to be working](https://github.com/argoproj/argo-workflows/issues/5376). If this is required, you need to build the argoexec container yourself by first adjusting the base image.
```

Comment:
```suggestion
- The `argoexec` Windows container is built using [nanoserver:1809](https://github.com/argoproj/argo-workflows/blob/master/Dockerfile.windows#L28) as the base image. Running a newer windows version (e.g. 1909) is currently [not confirmed to be working](https://github.com/argoproj/argo-workflows/issues/5376). If this is required, you need to build the `argoexec` container yourself by first adjusting the base image.
```

If we're going to link to the Dockerfile, can we use a permalink? The permalink should include the commit hash so that the document doesn't change under the link

Reply:
fixed 👍